### PR TITLE
Wip/andrej.picej@norik.com/bsp imx93 pd24.1.1

### PR DESCRIPTION
**Pull request by andrejpicej**
BSP manual for new upcoming i.MX93 PD24.1.1 release.

This release brings support for additional phyBOARD-Nash. Some peripherals differ between the two phyBOARD or are specific for phyBOARD Nash, this is noted and hopefully clear in the BSP manual.

TODO: still need to add phyBOARD Nash pictures with the last commit.

### CODE REVIEW

File: source/bsp/peripherals/emmc.rsti [hunk 1]
```diff
@@ -2,7 +2,7 @@ eMMC Devices
 ------------
 
 PHYTEC modules like phyCORE-|soc| is populated with an eMMC memory chip as the
-main storage. eMMC devices contain raw MLC memory cells combined with a memory
+main storage. eMMC devices contain raw MLC or TLC combined with a memory
```
Comment (tremmet) [inline source/bsp/peripherals/emmc.rsti:5]: Maybe write out the full description of MLC and TLC during the first usage and the acronym in bracket. Like done for BKOPS below.

File: source/bsp/peripherals/adc.rsti [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ADC
+---
+
+The PHYTEC |soc| include general purpose Analog-to-Digita Converters (ADC) which
```
Comment (tremmet) [inline source/bsp/peripherals/adc.rsti:4]: s/Analog-to-Digita/Analog-to-Digital/g

File: source/bsp/imx8/imx8mm/head.rst [hunk 3]
```diff
@@ -309,7 +309,7 @@ numbers of these UART units. UART1 can also be used as RS-485. For this,
 
            **UART1 RS232**
 
-.. include:: /bsp/imx8/peripherals/rs232-485.rsti
+.. include:: /bsp/peripherals/rs232-485.rsti
```
Comment (tremmet) [inline source/bsp/imx8/imx8mm/head.rst:312]: I agree that the documentation chapter is really generic and can be moved to a common place. But to have it really generic I would suggest to split the file. As I can image that not all boards have RS232 and RS485 available.

File: source/bsp/imx9/imx93/peripherals/tpm.rsti [hunk 4]
```diff
@@ -0,0 +1,57 @@
+TPM
+---
+
+The **phyBOARD-Nash** i.MX 93 is equipped with a Trusted Platform Module (TPM)
+that provides hardware-based security functions.
+
+Here are some useful stuff to do with TPM:
```
Comment (tremmet) [inline source/bsp/imx9/imx93/peripherals/tpm.rsti:7]: Maybe better write: "Here are some useful examples to work with the TPM"

**Comment by tremmet** [comment]
Commit message "bsp: peripherals: Add ADC chepter"  needs a typo fix.

**Review by tremmet — CHANGES_REQUESTED** [review]

File: source/bsp/peripherals/emmc.rsti (hunk 1 above)
Comment (andrejpicej) [reply source/bsp/peripherals/emmc.rsti]: Makes sense. Fixed.

File: source/bsp/peripherals/adc.rsti (hunk 2 above)
Comment (andrejpicej) [reply source/bsp/peripherals/adc.rsti]: Fixed.

File: source/bsp/imx8/imx8mm/head.rst (hunk 3 above)
Comment (andrejpicej) [reply source/bsp/imx8/imx8mm/head.rst]: Split and updated.

**Comment by andrejpicej** [comment]
> Commit message "bsp: peripherals: Add ADC chepter" needs a typo fix.

Fixed this and all other remarks. Thanks for your review :)

**Review by tremmet — APPROVED** [review]

**Review by BHahn42618 — APPROVED** [review]

**Comment by andrejpicej** [comment]
Rebased to main

**Review by pfiser — APPROVED** [review]

**Comment by andrejpicej** [comment]
@jonas-rem Can you merge this, please?

**Pull request merged**